cmd/config.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	guid "github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// configCmd represents the config command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Links uuid with Snyk token.",
+	Long: `config registers Snyk Token 
+	with UUID on crda server and outputs UUID.`,
+	Args: validateArgs,
+	Run:  main,
+}
+
+func init() {
+	rootCmd.AddCommand(configCmd)
+
+}
+func validateArgs(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("snyk token is missing")
+	}
+	return nil
+}
+
+func requestServer(synkToken string) string {
+	ThreeScale := viper.GetString("3scaleToken")
+	url := viper.GetString("server") + "/user?user_key=" + string(ThreeScale)
+	uuid := guid.New().String()
```

Comment:
You shouldn't be generating a UUID., actually the `/user` endpoints generates one and you need to persist from the cli and use it for further requests.

Reply:
ack.